index.html

```diff
@@ -110,7 +115,7 @@ <h3 class="card-title">Multi-Post Stories</h3>
                     </ul>
                   </div>
                    <div class="seeProject">
-                  <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+                  <a href="#"><button type="button" class="btn1" id="button_linkProj">See Project</button></a>  
                   </div>
                 </div>
             </div>
```

Comment:
- [ ] According to the requirements, The projects/work section should be created dynamically using js with the information from the projects array (all the HTML of the section should be created when the page loads). 

> So please, remove all the html code for this section (all the cards), and create them dynamically using js 👍


![image](https://user-images.githubusercontent.com/45231276/195539487-217d6c63-086b-4613-ac4a-cbba1e877b55.png)